client/app/visualizations/EditVisualizationDialog.jsx

```diff
@@ -13,7 +13,6 @@ import Visualization from "@/services/visualization";
 import recordEvent from "@/services/recordEvent";
 
 // ANGULAR_REMOVE_ME Remove when all visualizations will be migrated to React
```

Comment:
I think this comment should have been removed too, we can remove it during cleanup, but just to have in mind there's nothing else to be done.

Reply:
I wanted to remove it (and few other things) in this PR, but realized that it does not break the app, and decided to not add more changes to PR. I think it's better to do all the cleanup in own PR

Reply:
In this case I guess it's just the comment left :sweat_smile:, the "code part" that it was pointing to be removed seems to have been removed in this PR.